tests/unit/calendar/__snapshots__/calendar.normal.spec.js.snap

```diff
@@ -591,7 +591,7 @@ exports[`Calendar (normal mode) #render() should apply the new minDate on Calend
           onNavNextMonth={[Function]}
           onNavPreviousMonth={[Function]}
           onSelectDay={[Function]}
-          renderDate={2017-03-21T00:00:00.000Z}
+          renderDate={2017-03-23T00:00:00.000Z}
```

Comment:
no, it's because `new Date()` was not stub for Calendar's unit tests.
I created issue for that: https://github.com/Travix-International/travix-ui-kit/issues/48